docs/configuration.rst

```diff
@@ -221,6 +221,11 @@ These may include the following optional keys:
         Orphan fragments (those without an issue number) always have their content included.
         If a fragment was created, it means that information is important for end users.
 
+``check``
```

Comment:
I think that if this were ever needed, it would be easier to make the `check` a `dict|bool` rather than a separete knob from the `ignore_check`
```toml
[[tool.towncrier.type]]
...
check=false
[[tool.towncrier.type]]
...
check=true
[[tool.towncrier.type]]
...
check={"some_new_flag": "some_value"}
```

Reply:
```
check={"some_new_flag": "some_value"}
```

this looks bad to me... if we use this, why bother with .INI or .TOML and not just go with JSON  ?

If you dislike `ignore_check` then we can keep the current option.